nanoFramework.Device.Bluetooth/GenericAttributeProfile/GattPresentationFormat.cs

```diff
@@ -0,0 +1,69 @@
+﻿//
+// Copyright (c) .NET Foundation and Contributors
+// See LICENSE file in the project root for full license information.
+//
+
+namespace nanoFramework.Device.Bluetooth.GenericAttributeProfile
+{
+    /// <summary>
+    ///  Represents the value of a single Presentation Format GATT Descriptor.
+    /// </summary>
+    public sealed class GattPresentationFormat //: IGattPresentationFormat
```

Comment:
remove dead code or mention why it's there
And like for all the other classes, why sealed?

Reply:
Removed